### DISCUSSION TITLE
Where can I find more information about the modulus test examples?

### DISCUSSION THREAD

**Discussion by Juns-Cheng** [idaholab/moose#29371] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Hello, everyone. 
I am learning MOOSE (solid mechanics module) by going through the tutorial examples and test examples. But many examples are given without enough information. So I wonder if they can be found somewhere else. Things like what an example is about, what functional is tested, what result to expect, etc. 
Or where can I find another more detailed beginner's tutorial. 
Thanks in advance.

**Comment by GiudGiud** [comment]
Hello

The test are mostly for continuous integration purposes. A proficient moose user can read them easily. For example and getting started material for solid mechanics, see this page:
https://mooseframework.inl.gov/modules/solid_mechanics/tutorials/introduction/index.html
or navigate from this page before going into object documentation
https://mooseframework.inl.gov/modules/solid_mechanics/index.html
